### ISSUE TITLE
Netflix Eureka 2.x - WIKI says discontinued?

### ISSUE THREAD

**Issue by shyamrox** [Netflix/eureka#1514]
I see tags for Netflix Eureka 2.x and a release (2.0.1) made in June 2023,  however the WIKI says its discontinued.

![image](https://github.com/Netflix/eureka/assets/58111872/f209153a-dcde-43f6-8bfa-8cb5c2c33dd4)

![image](https://github.com/Netflix/eureka/assets/58111872/3ef53845-c890-4b9b-bea2-36f65a26d221)

Can anyone confirm the state of Netflix Eureka 2.x?  Is it discontinued?

Thanks!

**Comment by spencergibb** [comment]
The old 2.x, which was a re-architecture, is not supported and is now located here https://github.com/Netflix/eureka/tree/2.x-archive .

The 2.x branch https://github.com/Netflix/eureka/tree/2.x is new and supported, but it was only an upgrade to jakartaee and is wire compatible with 1.x

The wiki needs to be updated by the Netflix team

**Comment by fatboyi** [comment]
Does it mean the project still active and Netflix is still updating the Eureka ?

Thanks.

**Comment by ouaibsky** [comment]
@spencergibb 

Do you know the process to have PR merged into branch 2.x.

I have one open from many months, and no feedback #1540 

Maybe I missed something ?

Thanks
Ouaibsky.

**Comment by spencergibb** [comment]
I'm only a contributor to this repo, not a maintainer

**Comment by zhashuyu** [comment]
Does this v2 version support subscribe instance, like this,

![Image](https://github.com/user-attachments/assets/6ff4d35f-7ce9-4b47-9069-aa56f505a23a)

**Comment by spencergibb** [comment]
@zhashuyu no, anything proposed in the 2.x-archive branch has not been released and is not in the 2.x branch
